workflows/virtual-screening-blueprint/definition/main.nf

```diff
@@ -41,7 +38,7 @@ workflow {
     RunDiffdock(
```

Comment:
let's talk about this one. I think the parallelization needs to happen in chunks. and the params don't look high enough to confirm. so I think create an upstream channel that is chunked (between molmim and this input)